### DISCUSSION TITLE
Facing issue with future ready talent account

### DISCUSSION THREAD

**Discussion by Gunasekharippili** [github/india#1088] [Students]
By mistake I created a new FRT account but, I already have a old one which is deactivated. Here there is problem when I linking my credentials (ms-learn, github, LinkedIn) to new FRT account it's says credentials linked other account. So, I request you to reactivate my old account or remove my credentials linked old account. 

Old account mail id: igsekhar143@gmail.com
New account mail id: 21a35a0419@pragati.ac.in

I already mailed to [support@futurereadytalent.in] they didn't responded me. I request to sort my problem very fastly. # @ey-mentor1 @ey-mentor2 @ey-mentor3
![Old_account](https://user-images.githubusercontent.com/112553561/228639135-e1cfd0c3-e763-4249-b290-30212ebce4f0.jpg)
![New_account](https://user-images.githubusercontent.com/112553561/228639328-afbacbad-dce8-48bd-8303-4d64e84af92b.jpg)

**Comment by Karthikrusum** [comment]
hi bro same problem please tell me bro please.

**Comment by ganeshmucharla** [comment]
Hi, can we delete this future ready talent because what if anyone have two accounts
